source/jquery.flot.js

```diff
@@ -984,6 +987,25 @@ Licensed under the MIT license.
             executeHooks(hooks.bindEvents, [eventHolder]);
         }
 
+        function addEventHandler(event, handler, eventHolder, priority) {
+            var key = eventHolder + event;
```

Comment:
So including `eventHolder` in the key is just a way to (potentially) reduce the size of each individual event list?  Given our conversation, whether we keep or leave off the parameter, it doesn’t seem like it will actually be very effective in partitioning the lists — I would just use `event` as the key.

Reply:
No. It's a way of identifying a unique holder of events to register a specific event against. Two different divs could have the same event registered against it, which is totally valid, but I don't want to manage the priorities of events on two different divs. This is only meant to manage the priorities of handlers registered against the same event holder.

Reply:
Fair enough; but the code as written does not have any dependency on separating those lists.  It will work for a merged list, since each `eventData` knows which `eventHolder` to bind to (at the cost of some unnecessary event registrations).

I am just skeptical of relying on the stability of the `toString` implementation for any arbitrary `eventHolder` object that gets passed to this method.